### PR TITLE
Improve translatable strings

### PR DESCRIPTION
**Pull request by akirk**
This to address [Jenia's comments regarding Jetpack 3.7](https://i18n.wordpress.com/2015/08/21/jetpack-3-7-is-ready-for/#comment-9760).

### CODE REVIEW

**Comment by akirk** [comment]
I think the CLI part can be improved quite a bit. Requiring the user to type an English word is suboptimal. While I have added a comment for translators (please try to put yourself into the position of someone who can't read code and actually doesn't see the surrounding code) to not translate it, I think we could do a 1/2 selection type of thing.

Please select your option:
1) Yes, please go ahead. I know that modifying this option may disrupt my Jetpack connection.
2) Cancel

@jenial any better ideas? This is in the context of a script that runs in the terminal, so we have to ask the user to enter something on the keyboard.

**Comment by zinigor** [comment]
Thank you for fixing it! I like the numbers idea, I'm going to create a new issue about it for 3.8.

**Comment by akirk** [comment]
@zinigor Thanks. Just one thing, this issue still is a bug not an enhancement, the new issue will be an enhancement. I also created a label i18n.

File: modules/module-info.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ function grofiles_more_info_connected() { ?>
 		</a>
 	</div>
 
-	<h5><?php esc_html_e( "What&#8217;s a Hovercard?", 'jetpack' ) ?></h5>
+	<h5><?php esc_html_e( "What&#8217t;s a Hovercard?", 'jetpack' ) ?></h5>
```
Comment (zinigor) [inline modules/module-info.php]: I'm pretty sure that's not intended, is it?

Comment (akirk) [inline modules/module-info.php]: Weird, no. I didn't touch this. Must be a Github bug.

**Comment by zinigor** [comment]
Thanks, @akirk! I have added a missed space and fixed indentation up a bit. Looks good to me!

**Comment by jenial** [comment]
>any better ideas? This is in the context of a script that runs in the terminal, so we have to ask the user to enter something on the keyboard.

Yes, I saw the Jetpack CLI strings in the previous release and the translators had queries about those. I shared some screenshots. I agree that typing in English commands is suboptimal from the localization point of view, and numbers would be better, but maybe there is an even better solution.

**Pull request merged**